cmds/core/ip/ip_linux.go

```diff
@@ -151,8 +151,7 @@ func parseFlags(args []string, out io.Writer) (cmd, error) {
 		fs.PrintDefaults()
 	}
 
-	fs.Parse(unixflag.ArgsToGoArgs(args))
-
+	fs.Parse(unixflag.ArgsToGoArgs(args[1:]))
```

Comment:
can't you just use fs.Args() here?

Reply:
I think `fs.Args()` is only populated after `fs.Parse()` is called. 
If I replace args[1:] in this instance, the flags are not parsed correctly.

Reply:
yeah I meant on the line below, but you're doing that, so we're  good.